test_output.py

```
@@ -0,0 +1,101 @@
+def example_function():
+    """
+    Function with docstring containing imports:
+    import docstring_fake
+    from docstring_fake import example
+    """
+    # Use the imported items
+    result = utility_function()
+    obj = TestClass("test")
+
+    # Use some standard library functions
+    data = json.dumps({"result": result, "value": obj.get_value(), "time": str(datetime.now())})
```

Ruff (0.11.9)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)